Formula/cataclysm.rb

```diff
@@ -8,46 +8,43 @@ class Cataclysm < Formula
 
   bottle do
     cellar :any
+    rebuild 1
```

Comment:
`bottle` block shouldn't be touched. Assuming you're trying to bump the revision you want a `revision 1` underneath the `sha256` line.